<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ADIF to SQLITE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableView" name="view">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>675</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFileReadDb"/>
    <addaction name="actionFileWriteDb"/>
    <addaction name="separator"/>
    <addaction name="actionFileOpenDb"/>
    <addaction name="separator"/>
    <addaction name="actionFileDisplayDb"/>
    <addaction name="separator"/>
    <addaction name="actionFileCloseDb"/>
    <addaction name="separator"/>
    <addaction name="actionFileExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFileReadDb">
   <property name="text">
    <string>Read ADI to Database</string>
   </property>
  </action>
  <action name="actionFileWriteDb">
   <property name="text">
    <string>Write Database to ADI</string>
   </property>
  </action>
  <action name="actionFileExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFileCloseDb">
   <property name="text">
    <string>Close Database</string>
   </property>
  </action>
  <action name="actionFileDisplayDb">
   <property name="text">
    <string>Display Database</string>
   </property>
  </action>
  <action name="actionFileOpenDb">
   <property name="text">
    <string>Open Database</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
